### PR TITLE
Add script to check #1306 issue

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check [#1306](https://github.com/smartdevicelink/sdl_core/issues/1306)

This PR is **[ready]** for review.

### Summary
Add test scripts to check the 1306 issue.

### ATF version
develop

### Changelog
1. Added new script
2. Updated test set for `7.0` issues

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	local cid = self.mobileSession1:SendRPC("SetGlobalProperties", params.requestParams)
+
+	params.UiParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("UI.SetGlobalProperties", params.UiParams)
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	TtsParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("TTS.SetGlobalProperties", TtsParams)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: `TtsParams` is empty, must be `params.TtsParams`.

File: test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+local function setGlobalPropertiesSUCCES(params, self)
+	local cid = self.mobileSession1:SendRPC("SetGlobalProperties", params.requestParams)
+
+	params.UiParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("UI.SetGlobalProperties", params.UiParams)
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	TtsParams.appID = common.getHMIAppId()
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: see comment below

File: test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+local function ResetGlobalPropertiesSUCCES(params, self)
+	local cid = self.mobileSession1:SendRPC("SetGlobalProperties", params.requestParams)
+
+	params.UiParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("UI.SetGlobalProperties", params.UiParams)
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	TtsParams.appID = common.getHMIAppId()
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: `TtsParams` -> `params.TtsParams`

File: test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+	EXPECT_HMICALL("TTS.SetGlobalProperties", TtsParams)
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+	self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+	self.mobileSession1:ExpectNotification("OnHashChange")
+end
+
+local function ResetGlobalPropertiesSUCCES(params, self)
+	local cid = self.mobileSession1:SendRPC("SetGlobalProperties", params.requestParams)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: Request from mobile must be ResetGlobalProperties,
also pay attention that params for ResetGlobalProperties are not the same as for SetGlobalProperties,

File: test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+	params.UiParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("UI.SetGlobalProperties", params.UiParams)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: must be expected default values from .ini file

File: test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	TtsParams.appID = common.getHMIAppId()
+	EXPECT_HMICALL("TTS.SetGlobalProperties", TtsParams)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: must be expected default values from .ini file

File: test_scripts/Defects/5_0/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+	vrHelpTitle = requestParams.vrHelpTitle,
+	vrHelp = requestParams.vrHelp,
+}
+
+local TtsParams = {
+	timeoutPrompt = requestParams.timeoutPrompt,
+	helpPrompt = requestParams.helpPrompt
+}
+
+local allParams = {
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: The better usage:
```
local allParams = {
	requestParams = requestParams,
	UiParams = {
	        vrHelpTitle = requestParams.vrHelpTitle,
	        vrHelp = requestParams.vrHelp,
        },
	TtsParams = {
	       timeoutPrompt = requestParams.timeoutPrompt,
	       helpPrompt = requestParams.helpPrompt
        }
}

```

File: test_scripts/Defects/5_0/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+local requesUIparams = {
+	vrHelpTitle = getValueFromIniFile(pathToIniFile, "HelpTitle"),
+	vrHelp = {{ position = 1, text = actions.getConfigAppParams(pAppId).appName }}
+}
+
+local paramHP = getValueFromIniFile(pathToIniFile, "HelpPromt")
+local paramTP = getValueFromIniFile(pathToIniFile, "TimeOutPromt")
+
+local requesTTSparams = {
+	helpPrompt = {{ text = paramHP[1], type = "TEXT"}, {text = paramHP[2], type = "TEXT"}},
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua]: paramHP and paramTP can contain not 2 elements, need to get the array and set values for helpPrompt elements in loop.

**Comment by GetmanetsIrina** [comment]
Сurrent develop(a7db5cc) branch is merged into PR.
Script is applicable for all policy flows and transport types.
Script is moved to 6_1 folder.
Test set for script is added.
Some update has been performed in script.

**Review by dboltovskyi — APPROVED** [review]

**Review by iCollin — APPROVED** [review]

**Pull request merged**